.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.10
```

Comment:
It seems like the CI pipeline is failing since it cannot install python. It might be that there must be:

python-version: 3.10.0

but not sure, we would have to try.

Reply:
How can we test?